ci/build_python.sh

```diff
@@ -30,14 +30,13 @@ RAPIDS_PACKAGE_VERSION=$(head -1 ./VERSION) rapids-conda-retry build \
 
 sccache --show-adv-stats
 
-# Build cuml-cpu only in CUDA 11 jobs since it only depends on python
+# Build cuml-cpu only in CUDA 12 jobs since it only depends on python
 # version
 RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
-if [[ ${RAPIDS_CUDA_MAJOR} == "11" ]]; then
+if [[ ${RAPIDS_CUDA_MAJOR} == "12" ]]; then
```

Comment:
One less thing to change in a possible future where this project drops CUDA 11 support. This isn't strictly necessary for this PR but seemed small enough that it'd be ok to include here... let me know if you'd like it removed.